website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx

```diff
@@ -0,0 +1,302 @@
+---
+layout: docs
+page_title: Single Consul Datacenter in Multiple Kubernetes Clusters - Kubernetes
+description: Single Consul Datacenter deployed in multiple Kubernetes clusters
+---
+
+# Single Consul Datacenter in Multiple Kubernetes Clusters
+
+~> Requires consul-helm v0.33.0 or higher.
```

Comment:
```suggestion
~> Requires consul-helm v0.32.0 or higher.
```
Is this a typo?

Reply:
No this would require a new release of Consul Helm, our next version would be 0.33.0

Reply:
Ah that makes sense! my bad!

Reply:
should I change it to 0.32.1?

Reply:
I think that makes sense to go 0.32.1 given its a single commit release